### ISSUE TITLE
Adding confidence intervals to ggspider

### ISSUE THREAD

**Issue by TMalta** [Ringomed/ggvanced#2]
Hi @Ringomed ,

I am sorry to bother you with this, but I can't figure it out myself. Is it possible to add confidence intervals to your ggspider plots?
I was trying to get something on the same lines as the image below. 
![image](https://github.com/Ringomed/ggvanced/assets/8023683/f8095ced-9f6d-43a9-856b-6d15779c0917)

Best regards,
Tiago

**Comment by Ringomed** [comment]
Hi @TMalta, 

no easy way to do it with the current setup comes to mind, but I will implement it as an additional feature after the weekend.

Best,
Zvonimir

**Comment by Ringomed** [comment]
Hi, @TMalta ,

I have added the requested feature. However, I opted for only displaying the confidence itnervals along the axis lines, as the shaded area between the axis carries no additional information, but only increases the plot clutter.

I have also updated the vignette with examples utilizing the new features.

Best,
Zvonimir Boban

![image](https://github.com/Ringomed/ggvanced/assets/60142390/77f5f770-7e68-4a52-937c-48ca1f51ef23)
